scripts/sda-admin

```diff
@@ -0,0 +1,448 @@
+#!/bin/sh
+
+set -u -e
+
+myself=$0
+
+# Default values for global options.
+mq_credentials=test:test	# --mq-credentials user:pass
+mq_host=localhost		# --mq-host hostname
+sda_config=s3cmd.conf		# --sda-config pathname
+sda_key=crypt4gh_key.pub	# --sda-key pathname
```

Comment:
Should these also be setable as ENVs?
It would make life easier when ingesting many files/dataset at the same time.

Reply:
> Should these also be setable as ENVs? It would make life easier when ingesting many files/dataset at the same time.

Yes, this is easily done. I'll turn their names into uppercase at the same time to signal that they are initially taken from the environment if they are available there.